src/main/java/io/phasetwo/service/resource/OrganizationsResource.java

```diff
@@ -117,7 +117,7 @@ public Response acceptInvitation(@PathParam("invitationId") String invitationId)
         (session) -> {
           Invitations.memberFromInvitation(invitation, auth.getUser());
           invitation.getOrganization().revokeInvitation(invitationId);
-          EventBuilder event = new EventBuilder(realm, session, connection);
+          EventBuilder event = new EventBuilder(realm, this.session, connection);
```

Comment:
Actually, I dont think this will work. The `this.session` can be null. We should use the `runJobInTransaction` that takes a context.